x11-misc/dunst/dunst-9999.ebuild

```diff
@@ -12,21 +12,25 @@ HOMEPAGE="https://dunst-project.org/ https://github.com/dunst-project/dunst"
 
 LICENSE="BSD"
 SLOT="0"
-IUSE="wayland"
+IUSE="+X wayland +dunstify +completions +xdg"
 
 DEPEND="
 	dev-libs/glib:2
 	sys-apps/dbus
-	x11-libs/cairo[X,glib]
+	x11-libs/cairo[glib]
```

Comment:
Instead adding ```x11-libs/cairo[X]``` to ```X? (``` you could do instead: ```x11-libs/cairo[X?,glib]```, so you don't have to add twice the same dep with different use flags.

Reply:
thanks, didn't know about that :+1: